inc/views/base/search/results_posts_row.twig

```diff
@@ -0,0 +1,48 @@
+{% if post.visible == 0 %}
+    {% set row = 'trow_shaded' %}
+{% elseif post.visible == -1 %}
+    {% set row = 'trow_shaded trow_deleted' %}
+{% else %}
+    {% set row = alt_trow() %}
+{% endif %}
+<tr class="inline_row">
+    <td align="center" class="{{ row }}" width="2%"><span class="thread_status {{ post.folder }}" title="{{ post.folder_label }}">&nbsp;</span></td>
+    <td align="center" class="{{ row }}" width="2%">
+        {% if post.hasicon %}
+            <img src="{{ post.icon_path }}" alt="{{ post.icon_name }}" title="{{ post.icon_name }}" /> 
+        {% else %}
+            &nbsp;
+        {% endif %}
+    </td>
+    <td class="{{ row }}">
+        <span class="smalltext">
+            {{ lang.post_thread }} <a href="{{ post.thread_url|raw }}">{{ post.thread_subject }}</a><br />
+            {{ lang.post_subject }} <a href="{{ post.post_url|raw }}#pid{{ post.pid }}">
+                {% if post.subject|length > 50 %}
+                    {{ post.subject|slice(0, 50) }}...
+                {% else %}
+                    {{ post.subject }}
+                {% endif %}
+            </a>
+        </span><br />
+        <table width="100%"><tr><td><span class="smalltext"><em>
+            {% if post.message|length > 200 %}
```

Comment:
This logic can be simplified. The documentation from the `slice` filter states:

> If length is given and is positive, then the sequence will have up to that many elements in it. If the variable is shorter than the length, then only the available variable elements will be present. If length is given and is negative then the sequence will stop that many elements from the end of the variable. If it is omitted, then the sequence will have everything from offset up until the end of the variable.

Reply:
Changed it.